File: FlankSpeedLegacyDemo.Web/Models/Comment.cs
namespace FlankSpeedLegacyDemo.Web.Models;

public class Comment
{
    public int Id { get; set; }
    public int WorkItemId { get; set; }
    public WorkItem? WorkItem { get; set; }
    public int AuthorUserId { get; set; }
    public User? AuthorUser { get; set; }
    public string Text { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}

File: power-apps-migration/src/models/Comment.ts
/**
 * Migrated from: FlankSpeedLegacyDemo.Web/Models/Comment.cs
 * Source: Comments attached to WorkItems
 * Target: Azure SQL [dbo].[Comments] via Power Platform SQL connector
 */
import { User } from "./User";

export interface Comment {
  id: number;
  workItemId: number;
  authorUserId: number;
  authorUser?: User;
  text: string;
  createdAt: string;
}

export interface CommentFormData {
  workItemId: number;
  text: string;
}
